.ci/openjdk14-excluded.files

```diff
@@ -0,0 +1,514 @@
+<!-- This file generated by script at -->
+<!--     https://github.com/checkstyle/checkstyle/pull/8920#discussion_r513557040 -->
```

Comment:
please make it as comment in issue, it is more visible area.

all PR conversations become resolved "collabsed".

Reply:
Done.